packages/templating-tools/package.js

```diff
@@ -16,7 +16,7 @@ Package.onUse(function(api) {
     // boilerplate-generator uses spacebars-compiler.)
     // XXX maybe uglify should be applied by this plugin instead of via magic
     // weak dependency.
-    'minifier-js@1.2.14'
+    'minifier-js@1.2.14 || 2.0.0'
```

Comment:
This issue lies in Meteor will only install 1 version of each package at a time. Since we are bumping the version of `minifier-js` to `2.0.0`, meteor will fail to build because there is no package installed that matches those contraints because `@1.2.14` means anything 1.x.y where x is >= 2 and y >= 14, but that first number has to be 1. The new contraints I added, say x.y.z, where x = 1 or 2, if x is 1 refer to above. If x is 2, y and z must be >= 0. Ideally, we would just leave the version off, but according to the docs that is bad practice.